openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -325,7 +325,7 @@ def test_jwt_access_token_scopes_and_filters(self, grant_type):
         self.assert_valid_jwt_access_token(
             data['access_token'],
             self.user,
-            scopes,
+            data['scope'].split(' '),
```

Comment:
We might want an additional assertion against the original scopes, to ensure that all 3 match:
1. The scopes sent to the request.
2. The scopes in the response.
3. The scopes in the JWT.